source/templates/partials/_modal-test.twig

```diff
@@ -0,0 +1,9 @@
+{% extends "layouts/_modal.twig" %}
+
+{% set modal = {
+  title: 'This title de modal',
+  btnSubmit: 'Ajouter le projet',
+  btnReset: 'Reset'
```

Comment:
Don't write in english this data, in the design it's "Annuler" I think, check it